core/test/utils/assertions.hpp

```diff
@@ -290,7 +350,7 @@ ::testing::AssertionResult matrices_near(
  *                     first, second);
  * ```
  *
- * @see ASSERT_STRING_CONTAINS
+ * @see GKO_ASSERT_STRING_CONTAINS
```

Comment:
It is `GKO_ASSERT_STR_CONTAINS`, I think. [Line 399](https://github.com/ginkgo-project/ginkgo/blob/develop/core/test/utils/assertions.hpp#L399)

Reply:
True, I did only look at the missing `GKO_`, good catch!